### DISCUSSION TITLE
SD.next: broken for me (doesnt save meta into images sometimes, copy to 'from text' doesnt work etc)

### DISCUSSION THREAD

**Discussion by Tillerz** [vladmandic/sdnext#2077] [General]
I right now updated ( 0f10a9ce ), and this is not working:

- When I change the prompt and then render an image, the METADATA underneath the image doesn't get updated with the changes (they are saved to the image though when clicking SAVE).

- [EDIT] And now the images also do not save with the METADATA, something is weird here. I did a --reinstall, that didn't give any improvements.

- Settings / show all:

![img](https://i.gyazo.com/467db0e1fa2f296b52fbe86d3b988d84.png)

- It also looks like "Secondary sampler" doesn't get restored when you start the UI and click the button to load the data from previous session.

- Loading an image in "Process Image" and then sending to "From Text" doesn't fill the prompt and negative prompt. Uh, I think it doesn't send ANYTHING over to the other tab. **EDIT**: Ok, it does work for OLDER images, it does not work with the metadata of images I rendered today ( version d48a13a1 ).

**Comment by vladmandic** [comment]
New Gradio changed soo much, they don't care about backward compatibility. I'm fixing issues one-by-one as they come up.

Can you transfer this to issues?

**Reply by Tillerz** [reply]
https://github.com/vladmandic/automatic/issues/2078 :)
